openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -193,11 +193,18 @@ <h3 class="hd-6 section-title">${_("Course Handouts")}</h3>
 <%static:require_module_async module_name="js/commerce/track_ecommerce_events" class_name="TrackECommerceEvents">
 
   var personalizedLearnerSchedulesLink = $(".personalized_learner_schedules_button");
+  var sockLink = $("#sock");
 
     TrackECommerceEvents.trackUpsellClick(personalizedLearnerSchedulesLink, 'course_home_upgrade_shift_dates', {
       pageName: "course_home",
       linkType: "button",
       linkCategory: "personalized_learner_schedules"
     });
 
+    TrackECommerceEvents.trackUpsellClick(sockLink, 'course_home_sock', {
+      pageName: "course_home",
+      linkType: "button",
+      linkCategory: "green_upgrade"
```

Comment:
I noticed in other PRs the linkCategory seems to be the same as the ID, would keeping the same consistency be a good pattern?

Reply:
(see comments above regarding category, and this doc: https://docs.google.com/document/d/13Fwl3x-XUDGwm73ftkiKgiruiK3wWnlHeQ3mVSl4q_8/edit?ts=5f0cc156)